core/domain/rights_manager.py

```diff
@@ -1201,10 +1201,10 @@ def assign_role_for_exploration(
         new_role: str. The name of the new role: One of
             ROLE_OWNER
             ROLE_EDITOR
```

Comment:
Add comma at EOL, ditto above. Do this for lists, since now you're adding a period to the end.